src/plugins/condition/utils/operations.js

```diff
@@ -34,6 +34,10 @@ const convertToStrings = (input) => {
     return stringInputs;
 };
 
+const joinValues = (values, length) => {
```

Comment:
`const joinValues = (values = 1, length) => {` then can get rid of `1` in function call :P

Reply:
Yeah, but I want this to be explicitly defined. maybe the default should be the length of the values array. Let's leave it as is for now.